src/org/opendatakit/briefcase/util/FileSystemUtils.java

```diff
@@ -180,7 +219,7 @@ public static final boolean isODKInstancesParentFolder(File pathname) {
     return foi.exists() && foi.isDirectory();
   }
 
-
+  @SuppressWarnings("unchecked")
```

Comment:
I make it generic and I moved the supress warning to the loading from the file process.